Add Met Office installation method (#2751)
ESMValGroup/ESMValTool: doc/sphinx/source/quickstart/installation.rst

.. _install_on_hpc:

Pre-installed versions on HPC clusters / other servers
======================================================

ESMValTool is available on the HPC clusters CEDA-JASMIN and DKRZ-Levante, and on the Met Office Linux
estate, so there is no need to install ESMValTool if you are just running recipes:

 - CEDA-JASMIN: `esmvaltool` is available on the scientific compute nodes (`sciX.jasmin.ac.uk` where
   `X = 1, 2, 3, 4, 5`) after login and module loading via `module load esmvaltool`; see the helper page at
   `CEDA <https://help.jasmin.ac.uk/article/4955-community-software-esmvaltool>`__ ;
 - DKRZ-Levante: `esmvaltool` is available on login nodes (`levante.dkrz.de`) after login and module loading
   via `module load esmvaltool`; the command `module help esmvaltool` provides some information about the module.

>  - Met Office: `esmvaltool` is available on the Linux estate after login and module loading via `module load`;

Reviewer: You might want to use the exact command to make it more user-friendly
```suggestion
 - Met Office: `esmvaltool` is available on the Linux estate after login and module loading via `module load esmvaltool`;
```

Author: The command isn't actually 'module load esmvaltool'. We use a different slightly more complicated one that I haven't put here because it specifies the version and is subject to change because of that.

Reviewer: Makes sense, thanks for clarifying.

>    see the ESMValTool Community of Practice SharePoint site for more details.

Reviewer: Do you think it would be helpful to provide a link to this site? But I could imagine that such a link should not be publicly displayed.

Author: It's an internal link, so anyone who can access it generally knows how to already and I'm a bit cautious about showing internal links even if they can't be accessed.

.. _install_on_hpc:

Pre-installed versions on HPC clusters / other servers
======================================================

> The ESMValTool is also available on the HPC clusters CEDA-JASMIN and DKRZ-MISTRAL and there will be no need
> to install it yourself if you are just running diagnostics:

Reviewer 2: Adding the Met Office to the opening sentence, and correcting the name of the HPC cluster at DKRZ:

```suggestion
ESMValTool is available on the HPC clusters CEDA-JASMIN and DKRZ-Levante, and on the Met Office Linux
estate, so there is no need to install ESMValTool if you are just running recipes:
```

   `CEDA <https://help.jasmin.ac.uk/article/4955-community-software-esmvaltool>`__ ;
 - DKRZ-Levante: `esmvaltool` is available on login nodes (`levante.dkrz.de`) after login and module loading
   via `module load esmvaltool`; the command `module help esmvaltool` provides some information about the module.

>  - Met Office: `esmvaltool` is available on the linux estate after login and molude loading via `module load`; see the ESMValTool Community of Practice sharepoint site for more details.

Reviewer 2: ```suggestion
 - Met Office: `esmvaltool` is available on the Linux estate after login and module loading via `module load`;
   see the ESMValTool Community of Practice SharePoint site for more details.
```

Reviewer 2: Change introduced in 938ef17 :)
